core-java/src/main/java/org/spine3/server/aggregate/Aggregate.java

```diff
@@ -220,22 +225,31 @@ public void play(Iterable<EventRecord> records) throws InvocationTargetException
         }
     }
 
+    /**
+     * Applies events to an aggregate unless they are state-neutral.
+     *
+     * <p>Event applier should call {@link #incrementState(Message)},
```

Comment:
We should not rely on this. Please arrange 'automatic' version advancement. State neutral events should not advance the version. It is assumed that a state-neutral event comes in a sequence of events where first one modifies the state.